JavaScriptFileType.js

```diff
@@ -256,6 +256,25 @@ JavaScriptFileType.prototype.write = function(translations, locales) {
     } else {
         // generate mode
         resources = this.project.getTranslations(translationLocales);
+        this.customInherit = translationLocales.filter(function(locale){
+            return this.project.getLocaleInherit(locale) !== undefined;
+        }.bind(this));
+
+        if (this.customInherit.length > 0) {
+            this.customInherit.forEach(function(lo){
+                var res = this.project.getTranslations(lo);
+                if (res.length == 0) {
+                    var inheritlocale = this.project.getLocaleInherit(lo);
+                    var inheritlocaleRes = this.project.getTranslations([inheritlocale]);
+                    inheritlocaleRes.forEach(function(resource){
+                        var newres = resource.clone();
+                        newres.setTargetLocale(lo)
+                        file = resFileType.getResourceFile(lo);
+                        file.addResource(newres);
```

Comment:
Doesn't this get added to the file twice? (See lines 282,283 below)

Reply:
No, that's not adding the same resources twice.
Here's an example case:
localemap info is provided `--localeInherit en-AU:en-GB`,  and there's no en-AU.xliff file. In case the data should be generated by following en-GB's data.

But generate mode data is only from xliff. so I added code to create en-AU's resources by copying en-GB's data.
so line 273 is for adding en-AU, and line 282 is for en-GB